opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseDrawerInteractor.java

```diff
@@ -21,9 +26,14 @@
 
     private PlanDefinitionSearchRepository planDefinitionSearchRepository;
 
+    private RevealApplication application;
```

Comment:
Could we rename this variable to something more descriptive i.e `revealApplication`